src/components/ec-radio-btn/ec-radio-btn.vue

```diff
@@ -0,0 +1,359 @@
+<template>
+  <div
+    v-if="label"
+    data-test="ec-radio-btn__label"
+    class="ec-radio-btn__label"
+  >
+    {{ label }}
+  </div>
+  <div
+    data-test="ec-radio-btn__group"
+    class="ec-radio-btn__group"
+    :class="{'ec-radio-btn__group--is-single-line': isGroupInline}"
+  >
+    <div
+      v-for="(radio, radioIndex) in radioButtons"
+      :key="radioIndex"
+      class="ec-radio-btn"
+      :class="[
+        $attrs.class,
+        {
+          'tw-mt-16': radioIndex !== 0 && !isGroupInline,
+          'tw-ml-40': radioIndex !== 0 && isGroupInline,
+        }
+      ]"
+      :style="style"
+      :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-radio-btn ec-radio-btn-${radioIndex}` : `ec-radio-btn ec-radio-btn-${radioIndex}`"
+      @click.prevent.stop="onRadioBtnClick(radio.value)"
+    >
+      <input
+        v-bind="{
+          ...$attrs,
```

Comment:
why do we need this attrs here?

Reply:
i guess if we need to introduce other attributes from ebo

Reply:
Don't think we need it here, in input-field we used it for the placeholder for example, I don't like to be honest this approach.

Reply:
removed